sympy/integrals/tests/test_integrals.py

```diff
@@ -215,6 +215,10 @@ def test_issue_3560():
     assert integrate(1/sqrt(x)**3, x) == -2/sqrt(x)
 
 
+def test_issue_18038():
+    raises(AttributeError, lambda: integrate((x, x))
```

Comment:
One more parenthesis should be appended. Now a syntax error results.

Reply:
I made the changes but Travis is still showing some some error

Reply:
That was an internet connection issue. I have restarted the job.